test/utils/test-swagger-gen.ts

```diff
@@ -421,12 +426,17 @@ const generateSwaggers = async (
       ? "--disable-async-iterators=true"
       : "";
 
-    if (swagger.split("/").length === 1) {
+    if (swaggerOrConfig.split("/").length === 1) {
       // When given a filename look for it in test server, otherwise use the path
-      swaggerPath = `node_modules/@microsoft.azure/autorest.testserver/swagger/${swagger}`;
+      swaggerPath = `node_modules/@microsoft.azure/autorest.testserver/swagger/${swaggerOrConfig}`;
+    }
+
+    let inputFileCommand: string = `${swaggerPath}`;
+    if (swaggerPath.endsWith("json")) {

```

Comment:
It is possible to have swaggers with yml extension, you may want to check for that to future proof.